common/cards/card-plugins/effects/lightning-rod.js

```diff
@@ -0,0 +1,72 @@
+import {discardCard} from '../../../../server/utils'
+import {GameModel} from '../../../../server/models/game-model'
+import EffectCard from './_effect-card'
+import {getCardAtPos} from '../../../../server/utils/cards'
+
+/**
+ * @typedef {import('common/types/cards').CardPos} CardPos
+ */
+
+class LightningRodEffectCard extends EffectCard {
+	constructor() {
+		super({
+			id: 'lightning_rod',
+			name: 'Lightning Rod',
+			rarity: 'common',
+			description:
+				"Attach to any of your active or AFK Hermits.\n\n All damage done to you on your opponent's next turn is taken by the Hermit this card is attached to.\n\nDiscard after damage is taken.",
+		})
+	}
+
+	/**
+	 * @param {GameModel} game
+	 * @param {CardPos} pos
+	 */
+	canAttach(game, pos) {
```

Comment:
Why can't this be removed and the default used? isn't it identical? Everything else here looks good.

Reply:
It can be removed, I removed the restriction but I didn’t realize that without it this does the same as the default.